packages/material/scss/validator/_theme.scss

```diff
@@ -0,0 +1,3 @@
+@include exports("validator/theme") {
```

Comment:
@joneff 
Is exporting an empty module okay?

Reply:
sure